crypto/property/property_string.c

```diff
@@ -117,11 +117,10 @@ static PROPERTY_STRING *new_property_string(const char *s,
     PROPERTY_STRING *ps = OPENSSL_malloc(sizeof(*ps) + l);
 
     if (ps != NULL) {
-        memcpy(ps->body, s, l + 1);
```

Comment:
If the `s` does not need to be aligned, and it does not need to be IMO. You can avoid the extra allocation call even without having the char body[1] member in the struct.

Just:
```
void *mem = OPENSSL_malloc(sizeof(*ps) + l + 1);
PROPERTY_STRING *ps = mem;
char *body = mem;

body += sizeof(*ps);
memcpy(body, s, l + 1);
ps->s = body;
```

Reply:
I still don't like that trick.  If you want to close this PR and do that, go ahead.